src/sentry/silo/README.md

```diff
@@ -0,0 +1,86 @@
+# Running Sentry in Hybrid Cloud (Updated 10/2022)
+
+## Background on Silos
+
+HC Sentry requires running two seperate instances and having them communicate between one another. They are:
```

Comment:
Nit, but I would personally avoid the "HC" abbreviation in documentation. Acronyms do not enhance clarity.

Reply:
good call 👍